components/header.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import logo from '../public/aclima-logo.svg';
+import theme from '../utils/theme';
+
+export const Header: React.FC = () => {
+  return (
+    <div>
+      <Image src={logo} alt="Aclima logo" />
+      <h1>Example Prototype</h1>
+      <style jsx>{`
```

Comment:
I don't mind styled jsx! It's not quite as compact or pretty as styled components but I feel like it's a good compromise between plain old css and more developer friendly options.

Reply:
I'm excited to try it out!